src/Middleware/SocialAuthMiddleware.php

```diff
@@ -173,9 +173,9 @@ class SocialAuthMiddleware implements MiddlewareInterface, EventDispatcherInterf
     /**
      * Constructor.
      *
-     * @param array $config Configuration.
-     * @param \Cake\Event\EventManager|null $eventManager Event manager instance.
-     * @param \SocialConnect\Provider\Session\SessionInterface|null $session Session handler for SocialConnect Service
+     * @param array                                                 $config       Configuration.
```

Comment:
All these unnecessary whitespace changes to docblocks need to be reverted.

Reply:
your absolut right, no idea where this comes from. so i just run 
`composer run-script cs-fix`
before committing and corrected all whitespace changes that i saw, hope thats fine now. BTW, is there an official way how to format and adjust code style?

Reply:
Not sure `composer run-script cs-fix` messed up the formatting instead of fixing it. I will look into it later.

There are still lot of unnecessary formatting changes remaining. Instead of trying to fix them just drop the commits you made after running CS fixer. There was only one CS error before that which you could fix manually.